scripts/setup_gradient_workflow.py

```
@@ -0,0 +1,232 @@
+            "prepared-training-data", 
+            "trained-model-output",
+            "training-logs",
+            "evaluation-results",
+            "training-pipeline-artifacts"
+        ]
+        
+        print("📊 Creating required datasets...")
+        for dataset_name in datasets:
+            try:
+                cmd = ["gradient", "datasets", "create", "--name", dataset_name]
+                result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.12.2)

`subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)